sdk/communication/azure-communication-callingserver/README.md

```diff
@@ -0,0 +1,188 @@
+# Azure Communication CallingServer Service client library for Java
+
+This package contains a Java SDK for Azure Communication CallingServer Service.
+
+[Source code][source] | [Package (Maven)][package] | [API reference documentation][api_documentation]
+| [Product documentation][product_docs]
+
+## Getting started
+
+### Prerequisites
+
+- An Azure account with an active subscription. [Create an account for free](https://azure.microsoft.com/free/?WT.mc_id=A261C142F).
+- [Java Development Kit (JDK)](https://docs.microsoft.com/java/azure/jdk/?view=azure-java-stable) version 8 or above.
+- [Apache Maven](https://maven.apache.org/download.cgi).
+- A deployed Communication Services resource. You can use the [Azure Portal](https://docs.microsoft.com/azure/communication-services/quickstarts/create-communication-resource?tabs=windows&pivots=platform-azp) or the [Azure PowerShell](https://docs.microsoft.com/powershell/module/az.communication/new-azcommunicationservice) to set it up.
+
+### Include the package
+
+[//]: # ({x-version-update-start;com.azure:azure-communication-callingserver;current})
+```xml
+<dependency>
+    <groupId>com.azure</groupId>
+    <artifactId>azure-communication-callingserver</artifactId>
+    <version>1.0.0-beta.1</version>
+</dependency>
+```
+[//]: # ({x-version-update-end})
+
+## Key concepts
+
+At a high level the Azure Communication CallingServer API will support 2 kinds of scenarios:
+
+- In-call app: Contoso server app is a participant in the call.  
+
+- Out-call app: Contoso server app is not a participant in the call - Server app can subscribe to events for calls between specific users or even all users belonging to the ACS azure resource.  
+
+Based on if the Contoso app join a call or not, APIs can be divided into two categories:   
+
+- In-call APIs: Contoso app is one of the participant in a call. It can be applicable for app to person (A2P) or person to app (P2A) case, or multi-party/group calls that server apps joined as a participant to provide audio/prompt.  
+
+- Out-of-call APIs: Contoso app can invoke these set of APIs without joining a call. It is applicable for actions on P2P calls, A2P calls, P2A calls and group calls.  
+
+## Examples
+
+### Create CallClient
+
+Once you initialized a `CallClient` class, you can do the following server calling operations:
```

Comment:
If you look at the other READMEs for communication SDKs. We first have a separate section on how to `Authenticate the client` and then we go into the APIs you can call with the client. Please follow a similar pattern here. https://github.com/Azure/azure-sdk-for-java/blob/master/sdk/communication/azure-communication-sms/README.md

Reply:
'Authenticate the Client' section added